src/components/dashboard/main-content/course-enrollments/course-cards/email-settings/tests/EmailSettingsModal.test.jsx

```diff
@@ -23,28 +22,38 @@ describe('<EmailSettingsModal />', () => {
     // is initially `false` until the modal is opened, at which point it's
     // set to whatever the correct value is for that particular course run.
     // Setting the `hasEmailsEnabled` prop here simulates that behavior.
-    wrapper.setProps({
-      hasEmailsEnabled: true,
-    });
+    wrapper.rerender((
+      <EmailSettingsModal
+        onClose={() => {}}
+        courseRunId="my+course+key"
+        hasEmailsEnabled
+      />
+    ));
```

Comment:
[note] It's unfortunate we will need to `rerender()` here, but don't seem an immediate way around it without refactoring the underlying `EmailSettingsModal` itself, which is likely out of scope for this particular work to replace Enzyme. Just wanted to leave a note that, while OK to leave as is, this generally seems to be a code smell.